changelog.md

```diff
@@ -1,3 +1,76 @@
+<a name="5.1.0"></a>
+# 5.1.0 (2019-05-30)
+[Full Changelog](https://github.com/googlechrome/lighthouse/compare/v5.0.0...v5.1.0)
+
+## New contributors!
+
+Thanks to our new contributors 👽🐷🐰🐯🐻! 
+Beng Eu <beng@bengeu.com>
+David Boyne <david.boyne@bglgroup.co.uk>
+
+## Core
+
+* budget: throw error on duplicate budget types ([#8915](https://github.com/googlechrome/lighthouse/pull/8915))
+* config: split out config helpers ([#9003](https://github.com/googlechrome/lighthouse/pull/9003))
+* gather-runner: treat NO_FCP as a pageLoadError ([#8340](https://github.com/googlechrome/lighthouse/pull/8340))
+* gather-runner: always reset scroll position ([#8625](https://github.com/googlechrome/lighthouse/pull/8625))
+* image-elements: drop spritesheet logic ([#8940](https://github.com/googlechrome/lighthouse/pull/8940))
+* refactor gather-runner control flow ([#8964](https://github.com/googlechrome/lighthouse/pull/8964))
+* logging: change log messages for gathering and trace retrieval ([#9029](https://github.com/googlechrome/lighthouse/pull/9029))
+* perf: speed up getNodePath ([#9055](https://github.com/googlechrome/lighthouse/pull/9055))
+* performance-budget: add MP-FID as budget metric ([#8905](https://github.com/googlechrome/lighthouse/pull/8905))
+* runner: fix missing timing properties in the browser ([#9072](https://github.com/googlechrome/lighthouse/pull/9072))
+* tap-targets: don't exclude visible position absolute elements from audit ([#7778](https://github.com/googlechrome/lighthouse/pull/7778))
```

Comment:
can you throw
- `gather-runner: treat NO_FCP as a pageLoadError `
- `gather-runner: always reset scroll position`
- `logging: change log messages for gathering and trace retrieval`
- `tap-targets: don't exclude visible position absolute elements from audit`

at the top? They're probably most significant for most users

Reply:
oh, actually, the scroll one was reverted. Maybe don't put `gather-runner: always reset scroll position` up top and add a `(reverted)` or something so it's not confusing?